client/lib/plans/constants.js

```diff
@@ -1159,8 +1156,7 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'Priority Support' ),
 		getDescription: () =>
 			i18n.translate(
-				'High quality support to help you get your website up ' +
-					'and running and working how you want it.'
+				'Hands-on support to help you set up your site ' + 'exactly how you want it.'
```

Comment:
No need for '+' as the string fits in one line :)

Reply:
👍